src/System.Threading.Channels/src/Configurations.props

```diff
@@ -2,9 +2,9 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <PackageConfigurations>
+      netcoreapp;
```

Comment:
@ericstj, did I do this correctly?  build -allconfigurations builds successfully locally, but I'm not sure if there are other things I need to tweak.  Basically I want to be able to use new .NET Core 3.0 APIs in the implementation when a developer targets .NET Core 3.0.

Reply:
This will drop the netcoreapp2.1 implementation from the package, instead netcoreapp2.1 will get the netstandard implementation.  You have added a netcoreapp3.0 implementation to the package, so that part is correct.  Is this what you wanted?

Reply:
Thanks.  I'm not really sure why it had a netcoreapp2.1 implementation, as it doesn't look like it was specializing anything for .NET Core 2.1 APIs.  Unless I'm missing it, then, I think it's ok to drop the 2.1 implementation assembly.

Reply:
@weshaggard added in https://github.com/dotnet/corefx/pull/32014 not sure why, looking into it.

Reply:
Thanks.

Reply:
I think his intention was to maintain a 2.1 implementation in packages when he remapped netcoreapp from 2.1 -> 3.0.  If he didn't do this then packages would either drop their 2.1 implementation (and potentially lose 2.1 specialization).  I recall a conversation around this where we were discussing that its bad for packages to include the version less target group since that creates issues when we reversion the repo.

In the case of Pipelines it was probably a mistake that it was actually packing the 2.1 impl since it wasn't specialized, so I'm OK with dropping it.

Reply:
Thanks for looking into it.